azkaban-hadoop-security-plugin/src/main/java/azkaban/security/HadoopSecurityManager_H_2_0.java

```diff
@@ -402,11 +403,10 @@ private void getToken(final String userToProxy) throws InterruptedException,
     }
   }
 
-  private void cancelHiveToken(final Token<? extends TokenIdentifier> t,
-      final String userToProxy) throws HadoopSecurityManagerException {
+  private void cancelHiveToken(final Token<? extends TokenIdentifier> t)
+      throws HadoopSecurityManagerException {
```

Comment:
I agree this is off context. I tried the other way and I see myself making less changes overall and it's counter productive for me. So, I would rather choose to make the code better and wait a little longer for the review. I tend to switch to other things and often forget / feel lazy to create a refactor patch before/after my code changes. 

I don't want to make a whole bunch of changes as well but definitely the ones that come to my eye. This one, for example, has been lurking around for a long time and I haven't touched it because I wanted to make a refactor PR that never happened.